docs/setup_guides/hpc.rst

```diff
@@ -38,27 +41,27 @@ Create a new conda environment for our pangeo work:
 ::
 
     conda create -n pangeo -c conda-forge \
-        python=3.6 dask distributed xarray jupyterlab mpi4py
+        python=3.6 dask distributed xarray jupyterlab mpi4py dask_jobqueue
```

Comment:
Here are my notes, by no means comprehensive:

* At the moment the start-notebook script on Cheyenne use the token, not the password. The person responsible for that script is working to change that, but maybe you want to mention the option to use the token to your users, especially because of what is mentioned below.

> a hashed password that encodes the same information, but is safe to include in a publicly accessible config file
* This is incorrect. The hash will only slow down potential attackers, not stop them. The config file should not be publicly accessible. 

* I noticed that your job scripts use **ompthreads**. I think that is needed only if your code is explicitly using OpenMP, otherwise you may be better served by just spawning the threads normally. But then we also enter into the discussion about thread/task pinning (aka processor binding) and that's a long one....

Reply:
@davidedelvento - thanks for giving this a read through. 

> maybe you want to mention the option to use the token to your users, especially because of what is mentioned below.

Until we have something functional that works with Cheyenne/Conda, I'm keen to keep this generic and to add a link to the jupyter documentation on this subject. I've kept this as HPC agnostic as possible with the hope that sys admins from groups like CISL would provide documentation specific to each machine. 

> The hash will only slow down potential attackers, not stop them. The config file should not be publicly accessible.

I will update accordingly. 

> I noticed that your job scripts use ompthreads...

I don't think this matters but it would be great if some CISL folks could evaluate this programatically. I can remove the `ompthreads` piece for now.

Reply:
You're very welcome @jhamman !

[token]
What to mention or not is your choice, obviously. But note that this is not Cheyenne-specific: tokens are the default setting for jupyter notebook now and that's why CISL supported them before passwords.

[ompthreads]
Well, that depends on what you intend to do. Perhaps @kmpaul knows both that and the way that OpenMP or regular threads work with PBS Pro?